summarizer/src/test/java/opennlp/summarization/lexicalchaining/LexChainTest.java

```
@@ -101,7 +102,7 @@ void testGetRelation() {
       assertEquals(WordRelation.MED_RELATION, rel2.relation);
       assertEquals(WordRelation.MED_RELATION, rel3.relation);
     } catch (Exception e) {
-      e.printStackTrace();
+      fail(e.getLocalizedMessage());
```

:+1: or just let it raise the exception with the whole stack trace.